include/GL/glui.h

```diff
@@ -1587,40 +1654,39 @@ class GLUIAPI GLUI_EditText : public GLUI_Control
     int                 text_x_offset;
     int                 substring_start; /*substring that gets displayed in box*/
     int                 substring_end;  
-    int                 sel_start, sel_end;  /* current selection */
-    int                 num_periods;
```

Comment:
I'm always in good cheer. But sometimes you feel like the devil to me. Sweet talking but all the while time wasting. Anyway, my words aren't tinged with emotion. Just sometimes doubt.